### DISCUSSION TITLE
New Custom Non-Standard Vocabulary - Submission & Publishing ?

### DISCUSSION THREAD

**Discussion by DrUrwin** [OHDSI/Vocabulary-v5.0#997] [Q&A]
Dear All,
 
I am looking to get in contact with the OHDSI Vocabulary Team about the development and release of a custom non-standard vocabulary we're creating for a large research cohort.

If anyone could please help with whom to contact and how to initiate the discussion I'd be most grateful!

Many thanks in advance and I look forward to hearing from you.

**Comment by TinyRickC137** [comment]
Dear @DrUrwin 

Unfortunately, discussions were not the best place to contact us. 

I suggest you post on the forum if you are still interested, and come to the OHDSI Vocabularies working group, the upcoming calls can be found [here](https://ohdsi.org/upcoming-working-group-calls/).

We now have [community contribution guidelines](https://github.com/OHDSI/Vocabulary-v5.0/wiki/Community-contribution) that may help you with the instructions.
